specification/imds/data-plane/Microsoft.InstanceMetadataService/stable/2020-10-01/examples/GetLoadBalancerMetadata.json

```diff
@@ -0,0 +1,46 @@
+{
+  "parameters": {
+    "api-version": "2020-10-01",
+    "Metadata": "true"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "loadbalancer": {
+          "publicIpAddresses": [
+            {
+              "frontendIpAddress": "51.0.0.1",
+              "privateIpAddress": "10.1.0.5"
```

Comment:
The metadata description is: "List of the instance level public or private IP of VM.", but it looks like this field also returns the LB's frontend IPs? If we are listing both public + private IP addresses, can the field's name just be ipAddresses or something similarly generic instead?

Reply:
also, is the private IP here intentionally different from the private IP listed in the other fields?